### PR TITLE
KLS-1232: fix linting issues

### PR DESCRIPTION
**Pull request by lewis-coulson-dit**
- Adds djlint precommit hook.
- Fixes all linting errors

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [X] Jira ticket has the correct status.
- [X] A clear/description pull request messaged added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/templates/components/header_footer/footer_base.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
     <div class="copyright-container {% block global_footer_class %}{% endblock %}">
         <div>
             {% block global_footer_logo %}{% endblock %}
-            <p class="copy-right">&copy; Crown copyright {% now "Y" %}. All rights reserved.</p>
+            <p class="copy-right">© Crown copyright {% now "Y" %}. All rights reserved.</p>
```
Comment (davidu1975) [inline core/templates/components/header_footer/footer_base.html:60]: is this right ?

File: sso_profile/templates/enrolment/companies-house-company-search.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {% extends 'enrolment/wizard.html' %}
 {% load static from static %}
 {% block above_form %}
-    {% include 'enrolment/information_box_companies_type.html'  with is_in_companies_house=True %}
-    <div id="form-step-body-text" id="business-details-information-box">
+    {% include 'enrolment/information_box_companies_type.html' with is_in_companies_house=True %}
+    <div id="form-step-body-text">
```
Comment (davidu1975) [inline sso_profile/templates/enrolment/companies-house-company-search.html:5]: div had 2 id's? and it has chosen 1?

Comment (lewis-coulson-dit) [reply sso_profile/templates/enrolment/companies-house-company-search.html:5]: yea so I removed this manually. The first ID was the only valid one / one being used.

File: core/templates/components/header_footer/footer_base.html (hunk 1 above)
Comment (lewis-coulson-dit) [reply core/templates/components/header_footer/footer_base.html:60]: yep part of linting was to replace html entity with literals

File: exportplan/templates/exportplan/pdf_download.html [hunk 3]
```diff
@@ -14,10 +17,10 @@
         <div id="footer_content" class="text-align-right page-footer">
             <p class="body-s">
                 Created at great.gov.uk
-                &nbsp;&nbsp;&nbsp;&mdash;&nbsp;&nbsp;&nbsp;
+                &nbsp;&nbsp;&nbsp;–&nbsp;&nbsp;&nbsp;
                 {% timezone "Europe/London" %}
                 {% now "h:i A" %} {% now "jS F Y" %}
-                &nbsp;&nbsp;&nbsp;&mdash;&nbsp;&nbsp;&nbsp;
+                &nbsp;&nbsp;&nbsp;–&nbsp;&nbsp;&nbsp;
```
Comment (davidu1975) [inline exportplan/templates/exportplan/pdf_download.html:23]: is this correct ?

File: exportplan/templates/exportplan/includes/pdf/pdf_section_marketing_approach.html [hunk 4]
```diff
@@ -39,10 +39,10 @@ <h6 class="body-s pad-y-0">Languages</h6>
                         <td>Rural</td>
                         <tr>
                             <td>
-                                <strong>{% widthratio country_data.urban_rural_percentages.urban_percentage 1 100 %}&percnt;</strong>
+                                <strong>{% widthratio country_data.urban_rural_percentages.urban_percentage 1 100 %}%</strong>
                             </td>
                             <td>
-                                <strong>{% widthratio country_data.urban_rural_percentages.rural_percentage 1 100 %}&percnt;</strong>
+                                <strong>{% widthratio country_data.urban_rural_percentages.rural_percentage 1 100 %}%</strong>
```
Comment (davidu1975) [inline exportplan/templates/exportplan/includes/pdf/pdf_section_marketing_approach.html:45]: correct ?

**Review by davidu1975 — APPROVED** [review]
lgtm

**Pull request merged**